external/js/kyber/package.json

```diff
@@ -49,9 +49,9 @@
     "eslint-config-prettier": "^2.9.0",
     "eslint-plugin-prettier": "^2.5.0",
     "jasmine": "^3.3.1",
-    "jsdoc-to-markdown": "^4.0.1",
+    "jsdoc-to-markdown": "^5.0.0",
```

Comment:
You can actually remove this package. I just checked and it is not used. I would assume a leftover when I switched to typedoc.